attestation-service/docs/policy.md

```diff
@@ -0,0 +1,84 @@
+# Attestation Policy (Developing)
+
+CoCo AS provides a flexible policy support based on OPA to facilitate the customized verification rules.
```

Comment:
Is it still accurate to call refer to OPA or should it be "based on Rego"?

Reply:
Based on rego sounds good!